presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -45,13 +48,18 @@ def anonymize(self, engine_request: AnonymizerRequest) -> str:
             end_of_text = min(analyzer_result.end, last_replacement_point)
             self.__validate_position_over_text(analyzer_result, text_len)
             output_text = (
-                output_text[: analyzer_result.start]
-                + new_text
-                + output_text[end_of_text:]
+                    output_text[: analyzer_result.start]
+                    + new_text
+                    + output_text[end_of_text:]
             )
             last_replacement_point = analyzer_result.start
         return output_text
 
+    def anonymizers(self):
```

Comment:
get_anonymizers

Reply:
I got a reject from Omri about the "get_". He suggested the naming. @SharonHart